h2o-docs/src/product/performance-and-prediction.rst

```diff
@@ -1163,6 +1163,87 @@ Using the previously imported and split airlines dataset, run the following to r
     ks
     0.20072346203696562
 
+Computing Model Metrics
+'''''''''''''''''''''''
+
+The ``h2o.make_metrics`` function computes a model metrics object from given predicted values (target for regression; class-1 probabilities, binomial, or per-class probabilities for classification). 
```

Comment:
The ``h2o.make_metrics`` is R specific name, maybe only ``make_metrics`` is general enough, WDYT? :) 

```suggestion
The ``make_metrics`` function computes a model metrics object from given predicted values (target for regression; class-1 probabilities, binomial, or per-class probabilities for classification). 
```